presto-spi/src/main/java/com/facebook/presto/spi/ErrorType.java

```diff
@@ -13,10 +13,20 @@
  */
 package com.facebook.presto.spi;
 
+import com.facebook.drift.annotations.ThriftEnum;
+import com.facebook.drift.annotations.ThriftEnumValue;
+
+@ThriftEnum
```

Comment:
Similar here--please comment saying that the ordinal needs to be backwards compatible.

Reply:
I was wondering if it would make sense to do a refactoring to add a integer code as a constructor
something like  `USER_ERROR(0)` , `INTERNAL_ERROR(1)` and so on.

Reply:
done